modules/dag-l1/src/test/scala/org/tessellation/dag/l1/domain/snapshot/programs/SnapshotProcessorSuite.scala

```diff
@@ -821,14 +823,9 @@ object SnapshotProcessorSuite extends SimpleIOSuite with TransactionGenerator {
           expect.same(
             (processingResult, lastSnapshotAfter),
             (
-              Left(
-                UnexpectedCaseCheckingAlignment(
-                  snapshotHeight6,
-                  snapshotSubHeight0,
-                  snapshotOrdinal10,
-                  snapshotHeight6,
-                  snapshotSubHeight0,
-                  snapshotOrdinal12
+              Right(
```

Comment:
In the next PR we can remove wrapping in Either, as we don't expect an error anymore.